exec/InitState.hs

```diff
@@ -68,6 +71,28 @@ runBoostrap cfg action = do
         let runAction = mkRunAction (ActionEnv rp cfg logMsg)
         unNat (exceptToFail . runAction) action
 
+copyDir :: FilePath -> FilePath -> IO ()
+copyDir from to = unfoldTreeM dirTree (Left "") >>= copyTree
+  where
+    usefull = not . (`elem` [".", ".."])
```

Comment:
(I would also not have written these function to begin with, but just used `System.Process` to call the shell.  Copying files is what shells are reasonably good at.)